.github/workflows/dockerimage.yml

```diff
@@ -39,12 +46,14 @@ jobs:
 
       - name: Build Image
         run: |-
-          docker build . -t eu.gcr.io/dfuse-302310/dfuse-eosio:latest -t eu.gcr.io/dfuse-302310/dfuse-eosio:${{  github.ref_name }}
+          docker build . -t eu.gcr.io/dfuse-302310/dfuse-eosio:${{  RELEASE_VERSION }} \
+                --build-arg EOSIO_TAG=${{ EOSIO_VERSION }} \
+                --build-arg DEB_PKG=eosio-${{ env.EOSIO_VERSION }}.deb \
```

Comment:
I think `env.EOSIO_VERSION` should be `EOSIO_VERSION`, although it doesn't make difference.

Reply:
will update to make it consistent